sdk/lib/delayimp/delayimp.c

```diff
@@ -1,11 +1,9 @@
 /*
- * PROJECT:         ReactOS SDK Library
- * LICENSE:         LGPL, see LGPL.txt in top level directory.
- * FILE:            lib/sdk/delayimp/delayimp.c
- * PURPOSE:         Library for delay importing from dlls
- * PROGRAMMERS:     Timo Kreuzer <timo.kreuzer@reactos.org>
- *                  Mark Jansen
- *
+ * PROJECT:     ReactOS delayimport Library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of delayimport library
+ * COPYRIGHT:   Copyright 2009 Timo Kreuzer <timo.kreuzer@reactos.org>
+ *              Copyright 2016 Mark Jansen
```

Comment:
Add Mark's address.

Reply:
He didn't add it before, maybe he doesn't want it there.

Reply:
Odd.

Reply:
I don't think I had an address back then.